.travis.yml

```diff
@@ -2,31 +2,50 @@ language: scala
 scala:
   - 2.10.5
   - 2.11.7
+
 jdk:
   - oraclejdk7 # openJDK crashes sometimes
-sudo: false
+
+sudo: required
+
+dist: trusty
+
 cache:
   directories:
     - $HOME/.ivy2
     - $HOME/.sbt/launchers/
     - $HOME/.cache/spark-versions
+
 env:
   matrix:
-    - SPARK_VERSION=2.0.1 SPARK_BUILD="spark-2.0.1-bin-hadoop2.7" SPARK_BUILD_URL="http://d3kbcqa49mib13.cloudfront.net/spark-2.0.1-bin-hadoop2.7.tgz"
-    - SPARK_VERSION=1.6.2 SPARK_BUILD="spark-1.6.2-bin-hadoop2.6" SPARK_BUILD_URL="http://d3kbcqa49mib13.cloudfront.net/spark-1.6.2-bin-hadoop2.6.tgz"
+    - PYSPARK_PYTHON=python3 SPARK_VERSION=2.0.1 SPARK_BUILD="spark-2.0.1-bin-hadoop2.7" SPARK_BUILD_URL="http://d3kbcqa49mib13.cloudfront.net/spark-2.0.1-bin-hadoop2.7.tgz"
+    - PYSPARK_PYTHON=python3 SPARK_VERSION=1.6.2 SPARK_BUILD="spark-1.6.2-bin-hadoop2.6" SPARK_BUILD_URL="http://d3kbcqa49mib13.cloudfront.net/spark-1.6.2-bin-hadoop2.6.tgz"
+    - PYSPARK_PYTHON=python2 SPARK_VERSION=2.0.1 SPARK_BUILD="spark-2.0.1-bin-hadoop2.7" SPARK_BUILD_URL="http://d3kbcqa49mib13.cloudfront.net/spark-2.0.1-bin-hadoop2.7.tgz"
+    - PYSPARK_PYTHON=python2 SPARK_VERSION=1.6.2 SPARK_BUILD="spark-1.6.2-bin-hadoop2.6" SPARK_BUILD_URL="http://d3kbcqa49mib13.cloudfront.net/spark-1.6.2-bin-hadoop2.6.tgz"
+
+addons:
+  apt:
+    packages:
+     - python3
```

Comment:
It seems that Python3 should be preinstalled on trusty image: https://docs.travis-ci.com/user/trusty-ci-environment/#Python. Do we need it here?